posts/quality-checklist-before-audit.md

```diff
@@ -0,0 +1,323 @@
+At Zeppelin we help to protect the core infrastructure of open and
+decentralized applications. My team is in charge of the
+[security audits](https://zeppelin.solutions/security-audits/),
+we review tons of lines of code written by very smart developers for projects
+that will shake the very foundations of our society.
+
+Finding security vulnerabilities on this futuristic cypherpunk environment is
+super challenging and fun; but I think that security actually starts in a
+pretty boring and traditional place, full of the wisdom that our elders* have
+collected through millennia of developing software in community. So here I
+want to share our checklist for boring quality things to do before an audit.
+
+# ✔️ Choose a free software license
+
+Closed code is insecure. If the people using your project can't inspect it,
+study it, hack and experiment on it, there's no way it can be trusted. If you
+hold abusive control over your users, there's nothing stopping you (or somebody
+more powerful than you) from making them vulnerable.
```

Comment:
I wanted to leave that open. That's why I was talking about power. It can be vulnerable to backdoors, but also to many other tricks that you can do with power and control.
I'll move this conversation to google docs.